test/test_core.py

```diff
@@ -6314,9 +6314,12 @@ def test_unicode_js_library(self):
     # are having the desired effect.
     # This means that files open()'d by emscripten without an explicit encoding will
     # cause this test to file, hopefully catching any places where we forget to do this.
-    create_file('expect_fail.py', 'print(len(open(r"%s").read()))' % test_file('unicode_library.js'))
-    err = self.expect_fail([PYTHON, 'expect_fail.py'], expect_traceback=True)
-    self.assertContained('UnicodeDecodeError', err)
+
+    # On Windows when Unicode support is enabled, this test code does not fail.
+    if not (WINDOWS and self.run_process(['chcp'], stdout=PIPE, shell=True).stdout.strip() == 'Active code page: 65001'):
```

Comment:
What is 65001?  

Do you know why this doesn't fail?  Is the issue that `PYTHONUTF8` and/or `PYTHONCOERCECLOCALE` are not being honored under windows?

Reply:
Fair enough, I guess the PYTHONUTF8 trick we use here doesn't work on windows.